api/client.py

```diff
@@ -51,7 +55,8 @@ def listen(args):
     except KeyboardInterrupt as e:
         print(f"Stopping.")
     finally:
-        path = '/'.join(['unsubscribe', args.channel])
+        # Pass subscription ID to /unsubscribe endpoint URL
+        path = '/'.join(['unsubscribe', str(res.json()['id'])])
```

Comment:
The `res` variable is also used in the `while True` loop, so we can't use it in `finally` as it will probably be redefined when listening for events.  Using subscription_id in `finally` would be clearer.